### ISSUE TITLE
Add Documentation for Custom Components!

### ISSUE THREAD

**Issue by andrewlalis** [tomdenboon/tom-site#1]
https://github.com/tomdenboon/tom-site/blob/9c751381a23ce4906441000d4920476c356d3184/src/components/personCard/CustomButton1.js#L3

**Comment by tomdenboon** [comment]
Can't add documentation. For code you do not understand.

**Comment by andrewlalis** [comment]
That's true. But then you need to close the issue before people get upset.
